pymoto/modules/io.py

```diff
@@ -352,3 +356,69 @@ def _response(self, *args):
             f.write("\n")
 
         self.iter += 1
+
+
+class TransientToVTI(Module):
+    """ Writes transient response vectors to a Paraview VTI file
+
+    This module utilizes a series file to properly associate the correct time with each timestep:
+    https://gitlab.kitware.com/paraview/paraview/blob/v5.5.0/Documentation/release/ParaView-5.5.0.md#json-based-new-meta-file-format-for-series-added
+
+    See also: :attr:`DomainDefinition.write_to_vti()`
+
+    The size of the vectors should be a multiple of ``nel`` or ``nnodes``. Based on their size they are marked as
+    cell-data or point-data in the VTI file. For 2D data (size is equal to ``2*nnodes``), the z-dimension is padded
+    with zeros to have 3-dimensional data. Also, non-transient block-vectors of multiple dimensions
+    (*e.g.* ``(2, 3*nnodes)``) are accepted, which get the suffixed as ``_00``.
+
```

Comment:
I've tried to read the time data, but not sure how to use it in Paraview. I get `dat.xxxx.vti` files, like the other VTI writer module, and next to that there are folders with transient data. How do you use the data generated? Might be nice to add an explanation on that here.